python/wakapi_anyide/runner.py

```diff
@@ -117,6 +117,17 @@ async def heartbeat_task(env: Environment, queue: Queue[Event], watchers: Sequen
         else:
             raise Exception(f"Failed to send heartbeat: {response.status} {last_text}")
 
+def languageProcessor(env: Environment, filename: str):
```

Comment:
You don't need to check for AttributeError. Pydantic has already validated the language mapping.